Makefile

```diff
@@ -14,17 +14,13 @@ PARENT_IMG ?= $(shell source ci/config/env; echo $${PARENT_IMG})
 # proof of concepts.
 
 # Customizable cluster domain for deploying wherever we want
-# CLUSTER_DOMAIN ?= $(shell kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain')
-CLUSTER_DOMAIN_CMD = kubectl get dnses.config.openshift.io/cluster -o json | jq -r '.spec.baseDomain'
-CLUSTER_DOMAIN ?= $(shell $(CLUSTER_DOMAIN_CMD))
-# REALM ?= APPS.$(shell echo $(CLUSTER_DOMAIN) | tr  '[:lower:]' '[:upper:]')
-REALM_CMD = echo "APPS.$(shell echo $(shell $(CLUSTER_DOMAIN_CMD)) | tr  '[:lower:]' '[:upper:]')"
+INGRESS_DOMAIN_CMD = kubectl get ingresses.config/cluster -o jsonpath='{.spec.domain}'
```

Comment:
Command looks correct to me:

>  To display your default ingress domain, run the following command:
> ``oc get ingresses.config/cluster -o jsonpath={.spec.domain}``

https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html